pyfr/integrators/dual/controllers.py

```diff
@@ -41,22 +45,58 @@ class DualNoneController(BaseDualController):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self._niters = self.cfg.getint('solver-time-integrator', 'niters')
+        sect = 'solver-time-integrator'
+
+        self._innersteps = 0
+        self._idxprev = 0
+        self._maxniters = self.cfg.getint(sect, 'niters')
+        self._atol = self.cfg.getfloat(sect, 'atol', default=1e-6)
+        self._rtol = self.cfg.getfloat(sect, 'rtol', default=1e-6)
+        self._chkevery = self.cfg.getfloat(sect, 'chk', default=5)
```

Comment:
Break up some of these blocks.  Check every should be an int rather than a float.